codegens/kotlin-okhttp/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@postman/codegen-kotlin-okhttp",
+  "version": "0.0.1",
+  "description": "Converts postman request into kotlin ktor code snippet",
+  "com_postman_plugin": {
+    "type": "code_generator",
+    "lang": "kotlin",
+    "variant": "Ktor",
+    "syntax_mode": "java"
+  },
+  "main": "index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "scripts": {
+    "test": "node npm/test.js",
+    "test-lint": "node npm/test-lint.js",
+    "test-unit": "node npm/test-unit.js",
+    "test-newman": "node npm/test-newman.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": ""
+  },
+  "author": "Postman Labs <help@getpostman.com>",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/postmanlabs/code-generators/tree/master/codegens/kotlin-okhttp",
+  "dependencies": {},
```

Comment:
We'd need to add `lodash` here as a dependency.

Reply:
Added